schemas/documents/registry/creditClass.js

```diff
@@ -94,5 +94,15 @@ export default {
       name: 'icon',
       type: 'image',
     },
+    {
+      title: 'Credit Type Icon',
+      name: 'creditTypeIcon',
+      type: 'image',
+    },
+    {
+      title: 'Credit Generation Method Icon',
+      name: 'creditGenerationMethodIcon',
+      type: 'image',
+    },
```

Comment:
I believe instead we should create reusable documents, eg a creditType document, with the credit type abbrev and the icon because with the current model, where multiple credit classes have the same type, we just have to repeat the same creditTypeIcon everywhere

Reply:
That's a good point, will update.

Reply:
@flagrede done here https://github.com/regen-network/regen-sanity/pull/39/commits/750d515d763c45adee6b499028415228a27b5b5e
Note that `creditType` was already existing. I create one `creditGenerationMethod` in the same format.